tests/integration/loki_tester/config.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright 2021 Canonical Ltd
+# See LICENSE file for licensing details.
+
+options:
+  log-to:
+    default: NOOP
+    description: |
+      Where should the charm workload dump its logs? 
+      Supported: `syslog`, `loki`, `file`.
+      Also a comma-separated combination of these is supported, as well as
+      `ALL`, for all modes at once, and `NOOP` for 'do nothing'.
```

Comment:
I wanted to avoid complexity and having to catch values being None. The log.py script is expecting to be called with a string, and with a `NOOP` it will do nothing. I'd rather have this than making the charm more complex and conditionally dropping a pebble layer. 
My goal was to keep the tester charm dumb and simple because it felt pointless to have it become so complex to need tests of its own :D